apps/web/src/routes/+layout.svelte

```diff
@@ -1,25 +1,75 @@
-<script>
+<script lang="ts">
 	import Sidebar from '$lib/components/sidebar/Sidebar.svelte';
 	import { fly } from 'svelte/transition';
 	import '../app.css';
 	import toast, { Toaster } from 'svelte-french-toast';
 	import Scans from '$lib/components/scans/Scans.svelte';
 	import { browser } from '$app/environment';
+	import { Focus } from 'lucide-svelte';
+	import { onMount } from 'svelte';
+	import { api } from '$lib/api/client';
+
+	let Device: any;
+
+	async function handleImageInput(event: InputEvent) {
+		console.log(event);
+		const formData = new FormData();
+		formData.append('img', event.target.files[0]);
+		formData.append('week_number', '4');
+		await fetch('http://localhost:5003/scan', {
```

Comment:
Na tohle můžeš použít ten vygvenerovaný klient

Reply:
S tim klientem jsem měl celkem problem. Pravděpodobně je to navedený na /api/scan místo /scan, ale idk

Reply:
Hm ok tak to necháme takhle a já potom asi trochu ředěláme ten backend a ty openAPI definice